_about_livecoms/04_donations.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: single
+sidebar:
+  nav: donations.md
+title: Donating to LiveCoMS 
+excerpt: How to donate to LiveCoMS
+permalink: /about/donations
+---
+
+### Donations to LiveCoMS
+
+  We very much appreciate financial donations to LiveCoMS! Donations
+  are made to the CU Foundation of University of Colorado Boulder, the
+  official publisher of the journal, to a fund exclusively for use of
+  LiveCoMS and controlled by the managing editors. The CU Foundation
+  is a 501(c)(3) charitable organization, and donations are thus
+  usually tax-deductible.
+ 
+Follow the link felow to donate:
+
+  <p><a href="https://giving.cu.edu/fund/living-journal-computational-molecular-science-support-fund" class="btn ">Donate Now to LiveCoMS!</a></p>
+
+You can also support LiveCoMS by:
+  * Contributing to LiveCoMS articles.
+  * Expressing your interest in becoming a reviewer ([early career](/about/earlycareer) or otherwise).
+  * Providing feedback on GitHub repositories for published articles to improve later versions of LiveCoMS articles.
+  * Citing LiveCoMS articles that you use in your research.
+  * Spreading the word about LiveCoMS to your colleagues, mentors, and students.
+
+
+### Financial costs for LiveCoMS
+
+LiveCoMS runs a very lean budget in order to keep costs for
+contributors low.  Managing editors and editorial board members
+receive no money or reimbursement from LiveCoMS.  Our current costs
+are:
+
+* $1200/year for journal hosting by [Scholastica](http://www.scholasticahq.com).
+* $250/year for reviewer services by [Scholastica](http://www.scholasticahq.com).
+* $504/year for Google Suites hosting of email and collaborative office software
+
+The main sources of income for LiveCoMS are donations and article
+charges. We in particular appreciate seed funding provided by [CU
+Boulder Libraries](https://www.colorado.edu/libraries/), as well as continuing in-kind contributions by
+managing DOI submissions and journal metadata. Article charges are
```

Comment:
Maybe "continuing support in terms of management of DOIs and journal metadata" would be clearer; "in-kind contributions ... by..." sounds awkward.

Reply:
Done.